tools/generators/migrate-converged-pkg/index.spec.ts

```diff
@@ -291,7 +374,7 @@ describe('migrate-converged-pkg generator', () => {
         preset: '../../jest.preset.js',
         globals: {
         'ts-jest': {
-        tsConfig: '<rootDir>/tsconfig.json',
+        tsConfig: '<rootDir>/tsconfig.spec.json',
```

Comment:
since our test extensions are `_.test.tsx` should we use `tsconfig.test.json` instead ?

```suggestion
        tsConfig: '<rootDir>/tsconfig.test.json',
```

Reply:
We can do that, although we use `spec` as well. My aim was to follow same patterns like community uses (nx) and to mirror what was agreed in RFC.  

in general whenever we'll come to naming agreement which will be ratified as part of contributing guidelines and generators we can change it to anything basically without big effort. WDYT?

Sidenote: 
- using `.test.*` doesn't make much sense as we are using behaviour driven approach `describe/it` etc
- I tried to address this long time ago without success/lack of understanding  (https://github.com/microsoft/fluentui/pull/16870/files#diff-cd6a8ca4fbad83f1a14b9277bc20c3261f3eb9a00685e52a485f9fd5ef822ab3R111)